Tests/SwiftKueryTests/TestSchema.swift

```diff
@@ -137,25 +137,30 @@ class TestSchema: XCTestCase {
         expectedCreateStmt = "CREATE TABLE \"table1\" (\"a\" text PRIMARY KEY DEFAULT 'qiwi' COLLATE \"en_US\", \"b\" integer AUTO_INCREMENT, \"c\" double DEFAULT 4.95 CHECK (\"c\" > 0), FOREIGN KEY (\"a\", \"b\") REFERENCES \"table2\"(\"b\", \"a\"))"
         XCTAssertEqual(createStmt, expectedCreateStmt, "\nError in table creation: \n\(createStmt) \ninstead of \n\(expectedCreateStmt)")
     }
-    
+
     func testCreateTable () {
         let connection = createConnection()
-        
+
         var t1 = Table1()
         var createStmt = createTable(t1, connection: connection)
         var expectedCreateStmt = "CREATE TABLE \"table1\" (\"a\" text PRIMARY KEY DEFAULT 'qiwi' COLLATE \"en_US\", \"b\" integer AUTO_INCREMENT, \"c\" double DEFAULT 4.95 CHECK (\"c\" > 0))"
         XCTAssertEqual(createStmt, expectedCreateStmt, "\nError in table creation: \n\(createStmt) \ninstead of \n\(expectedCreateStmt)")
-        
+
+        let t2 = Table1(name: "foobar")
```

Comment:
Your table name conflicts with a later declaration so the tests do not compile. Can you update the table name and then run 'swift test' to check it all passes please?